packages/block-library/src/quote/block.json

```diff
@@ -8,19 +8,11 @@
 	"keywords": [ "blockquote", "cite" ],
 	"textdomain": "default",
 	"attributes": {
-		"value": {
+		"attribution": {
 			"type": "string",
 			"source": "html",
-			"selector": "blockquote",
-			"multiline": "p",
-			"default": "",
-			"__experimentalRole": "content"
-		},
-		"citation": {
-			"type": "string",
-			"source": "html",
-			"selector": "cite",
-			"default": "",
+			"selector": "figcaption",
+			"default": null,
```

Comment:
Should this be `null`? Was `""` previously.

Reply:
Semantically, a `null` attribution seems to be more accurate to the front-end output (no figcaption) than what an empty-string attribution would imply.

Reply:
Do we have to provide a default value at all? Maybe we can just omit it. I thought the reason why we pass an empty string in so many is that we'd need it be a string for some reason.